source/templates/committer-welcome.txt

```diff
@@ -0,0 +1,67 @@
+To: private@[PROJECT].a.o, NewCommitterID@apache.org
+Subject: Welcome, New Committer!
```

Comment:
```suggestion
Subject: Welcome, [FIRST NAME] New Committer!
```

It would be great to at least have the first name in the subject to distinguish invitations of different people - it's more personal this way and then you can easily find the right email in the list of them and they will not be joined in a single thread (some email clients wll do it) if you send invitation to several people close to each other..